themes/theme-next/src/shared/navbar.ts

```diff
@@ -0,0 +1,93 @@
+export type NavItem = NavItemWithChildren | NavItemWithLink | string
+
+export interface NavItemWithLink {
+  text: string
+  link: string
+
+  /**
+   * A Regexp string, matching path will be active
+   *
+   * It's expected to be a regex string as RegExp object isn't serializable
+   *
+   * 正则表达式字符串，匹配的路径将被激活
+   *
+   * 由于 RegExp 对象不可序列化，因此我们需要将其定义为字符串。
+   */
+  activeMatch?: string
+
+  rel?: string
+  target?: string
+  noIcon?: boolean
```

Comment:
RFC here, is this needed? I am thinking that providing options like `breadcrumbIcon` `navbarIcon` `sidebarIcon` is enough, is there a common need for people to disable a page icon in all placces?

Reply:
`noIcon` is used to control whether to display the external link icon. Since this is a prop of the `VPLink` component, it has also been made configurable here.

Reply:
Maybe some jsdocs could be added here?